### DISCUSSION TITLE
Welcome to TestRocket Discussions!

### DISCUSSION THREAD

**Discussion by JohnPKosh** [JohnPKosh/TestRocket#25] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JohnPKosh** [comment]
Ok, so here we go people lets try out the new GitHub discussions! Anyone have anything to discuss?????? 

It is 8:00 am here and I have not had any coffee yet so I am drawing a blank!

**Reply by erxdkh** [reply]
Hallo, die Erde ist da.

**Reply by itadapter** [reply]
Gutten tag! Wie gehts?

**Reply by JohnPKosh** [reply]
I am doing well! Yes, I saw the issue notes. So indexing (per page boundary is still possible?) for LAR. So long as the page is a reasonable size and you seek over indexing by page you would skip needing to read the actual page data. That does bring up all kinds of decisions on what to index etc. Definitely worth much more discussion I think on how to approach.
